core/src/main/scala/org/apache/spark/sql/execution/CoprocessorRDD.scala

```diff
@@ -75,8 +75,13 @@ trait LeafExecRDD extends LeafExecNode {
       b.append(s"with dag request: $dagRequest")
       b.toString()
     } else {
-      s"TiDB $nodeName{$dagRequest}" +
-        s"${TiUtil.getReqEstCountStr(dagRequest)}"
+      if (dagRequest.getUseTiFlash) {
```

Comment:
If the difference is just TiFlash/TiKV, we can preserve the rest to avoid duplicate code.